src/helpers/searchParam.ts

```diff
@@ -7,7 +7,7 @@ import { SearchParams, SearchParamAsset, StringObject } from "types/types.d";
 
 const update = (
   param: SearchParams,
-  value: string,
+  value: string | string[],
```

Comment:
I don't remember exactly what, but I think I ran into some issues when using arrays in URL params. My idea was that we'd pass comma-separated strings here. That didn't work for this case?

Reply:
Good idea, I hadn't thought about that! I'll try to revert these changes and make the PR shorter...

Reply:
Done in https://github.com/stellar/stellar-demo-wallet/pull/206/commits/c89e33faf5c9a4e84945ca0ddaf02d05b7e6c838.